torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -279,33 +279,26 @@ def _affine_bounding_box_xyxy(
     bounding_box: torch.Tensor,
     image_size: Tuple[int, int],
     angle: float,
-    translate: Optional[List[float]] = None,
-    scale: Optional[float] = None,
-    shear: Optional[List[float]] = None,
+    translate: List[float],
+    scale: float,
+    shear: List[float],
     center: Optional[List[float]] = None,
     expand: bool = False,
 ) -> torch.Tensor:
-    dtype = bounding_box.dtype if torch.is_floating_point(bounding_box) else torch.float32
-    device = bounding_box.device
-
-    if translate is None:
-        translate = [0.0, 0.0]
-
-    if scale is None:
-        scale = 1.0
-
-    if shear is None:
-        shear = [0.0, 0.0]
+    # This is just a dummy value to avoid raising an error in `_affine_parse_args` although we don't have an
+    # interpolation mode for bounding boxes.
+    interpolation = InterpolationMode.NEAREST
+    angle, translate, shear, center = _affine_parse_args(angle, translate, scale, shear, interpolation, center)
```

Comment:
I have explicitly avoided calling this method on my other PR. See https://github.com/pytorch/vision/pull/6636#discussion_r978920646

Reply:
Well, I used it, since the old parsing, i.e. 

https://github.com/pytorch/vision/blob/2d92728341bbd3dc1e0f1e86c6a436049bbb3403/torchvision/prototype/transforms/functional/_geometry.py#L291-L306

failed for some of the new sample inputs. Meaning, the bounding box kernel did not support some parameters that the other kernels supported. This is why I used this to make sure that every kernel does the same. If you have speed concerns, I can look into what exactly caused the failure and only fix this.

Reply:
I've reverted the changes in 6dfc9657ce89fe9e018a11ee25a8e26c7d3d43c6 and added skips for the tests. They should actually be xfails, but the test framework currently doesn't support that. #6653 adds support for xfails. @datumbox you need to remove the skips in your PR, because the tests otherwise will not start to fail, even if you did something wrong.